hyperledger-student-management.yaml

```
@@ -0,0 +1,232 @@
+openapi: "3.0.0"
+info:
+  description: "This is the Student API for UC4's hyperledger component."
+  version: "0.3.3"
+  title: "UC4"
+servers:
+  - url: http://localhost:9000/hyperledger-student-management
```

I think the hyperledger course management service is on endpoint
hl-course-management, so I would suggest keeping this the same

correct me, if I am wrong @david-buderus 
```suggestion
  - url: http://localhost:9000/hl-student-management
```

---

There is no need for a new endpoint at all
As long we dont want to save the whole student on the chain